EUROTHRM/EUROTHRM-IOC-01App/Db/devEurothrm.db

```diff
@@ -146,13 +146,16 @@ record(ao, "$(P)CURRENT_TEMP")
 	field(DTYP, "asynFloat64")
 	field(OUT, "@asyn($(READ),0,1)CUR")
 	field(EGU, "K")
+    field(FLNK, "$(P)SENSOR_CHECK")
```

Comment:
As mentioned on `SENSOR_CHECK`, `CP` would potentially be neater to avoid the `FANOUT`